<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeEditDialog</class>
 <widget class="QDialog" name="ModeEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModeEditDialog</string>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbMode">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>420</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Mode</string>
   </property>
   <widget class="QRadioButton" name="rbWholeBlood">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Whole Blood</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPeripheral">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Peripheral</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPrediluent">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Prediluent</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
